data/transactions/logic/eval_test.go

```diff
@@ -4531,7 +4533,7 @@ func TestBits(t *testing.T) {
 
 	testAccepts(t, "byte 0xfffff0; int 21; int 1; setbit; byte 0xfffff4; ==", 3)
 	testAccepts(t, "byte 0xfffff4; int 1; int 0; setbit; byte 0xbffff4; ==", 3)
-	testPanics(t, "byte 0xfffff4; int 24; int 0; setbit; byte 0xbf; ==", 3)
+	testPanics(t, "byte 0xfffff4; int 24; int 0; setbit; byte 0xbffff4; ==", 3)
```

Comment:
I thought `typeEquals` only cares if the avmType is correct.  This change implies it does some bounds checking?